src/pages/workspace/WorkspaceProfilePage.tsx

```diff
@@ -57,6 +57,21 @@ function WorkspaceProfilePage({policy, currencyList = {}, route}: WorkSpaceProfi
     const readOnly = !PolicyUtils.isPolicyAdmin(policy);
     const imageStyle: StyleProp<ImageStyle> = isSmallScreenWidth ? [styles.mhv12, styles.mhn5] : [styles.mhv8, styles.mhn8];
 
+    const DefaultAvatar = useCallback(
+        () => (
+            <Avatar
+                containerStyles={styles.avatarXLarge}
+                imageStyles={[styles.avatarXLarge, styles.alignSelfCenter]}
+                source={policy?.avatar ?? ReportUtils.getDefaultWorkspaceAvatar(policyName)}
```

Comment:
This caused regression. `policy.avatar` can be empty string

<img width="712" alt="Screenshot 2024-02-29 at 2 37 01 AM" src="https://github.com/Expensify/App/assets/108292595/67a1fede-113b-40ea-86ea-efdb4423c27b">

Reply:
Here's the link to the issue: https://github.com/Expensify/App/issues/37465. It's now blocker btw